### ISSUE TITLE
[Tag]: A11y-Tag fails to meet 4:5:1 contrast ratio

### ISSUE THREAD

**Issue by dp1704** [SAP/ui5-webcomponents-react#7750]
### Describe the bug

[Tag]: A11y-Tag fails to meet 4:5:1 contrast ratio

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/data-display-tag--all-colors

Reproduction steps
examine the foreground color for the design Tag Negative
examine the foreground color for the design Tag Critical
compute the color contrast ratio

Expected Behaviour
Expected ratio should be at least 4:5:1.

Screenshot:

![Image](https://github.com/user-attachments/assets/b8db6048-936c-47ed-ad7e-535da3bcd771)

### Reproduction steps

1.
2.
3.
...

### UI5 Web Components for React Version

2.14.1

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Operating System

Mac OS

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @dp1704 

I will close this issue as there is already another one covering this topic. If you have any questions or more information, please address them in the [original issue](https://github.com/UI5/webcomponents/issues/11531).

**Comment by dp1704** [comment]
Hello @Lukas742 , there is no resolution on this original issue. please do not close it unless there is some resolution: https://github.com/UI5/webcomponents/issues/11531. please provide an update on the fix. this is impacting lot of customers for the accessibility standpoint.

Thank you,
Dhruti

**Comment by Lukas742** [comment]
Hi @dp1704 

this is a duplicate. Please raise any questions or concerns in the original issue, because then the responsible team can check. The `Tag` component is not developed in this repo.
